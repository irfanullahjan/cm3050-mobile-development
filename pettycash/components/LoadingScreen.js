import { ActivityIndicator, View } from "react-native";

export function LoadingScreen() {
  return (
    <View style={{ flex: 1, alignContent: "center", justifyContent: "center" }}>
      <ActivityIndicator />
    </View>
  );
}
