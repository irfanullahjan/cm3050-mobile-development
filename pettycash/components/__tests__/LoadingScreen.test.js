import { render } from "@testing-library/react-native";
import React from "react";

import { LoadingScreen } from "../LoadingScreen";

describe("LoadingScreen", () => {
  it("renders correctly", () => {
    const tree = render(<LoadingScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
