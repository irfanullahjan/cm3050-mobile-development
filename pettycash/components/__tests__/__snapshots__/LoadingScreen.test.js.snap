// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen renders correctly 1`] = `
<View
  style={
    Object {
      "alignContent": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator />
</View>
`;
